.ci/go-test.sh

```diff
@@ -240,8 +240,11 @@ main()
 
 	[ -z "$test_packages" ] && echo "INFO: no golang code to test" && exit 0
 
+	local go_ldflags
+	[ "$(uname -m)" = s390x ] && go_ldflags="-extldflags -Wl,--s390-pgste"
+
 	# KATA_GO_TEST_FLAGS can be set to change the flags passed to "go test".
-	go_test_flags=${KATA_GO_TEST_FLAGS:-"-v $race -timeout $timeout_value"}
+	go_test_flags=${KATA_GO_TEST_FLAGS:-"-v $race -timeout $timeout_value -ldflags '$go_ldflags'"}
```

Comment:
can `ldflags` be empty ? :thinking:

Reply:
yes, you can build with `-ldflags ''` without any consequences AFAICT (we will know with the tests for sure)

Reply:
we do need to escape because of `eval` though (updated)